Source/WebCore/rendering/style/RenderStyleSetters.h

```diff
@@ -508,6 +508,11 @@ inline void RenderStyle::setOrphans(unsigned short count)
     SET_PAIR(m_rareInheritedData, orphans, clampedCount, hasAutoOrphans, false);
 }
 
+inline void RenderStyle::setScrollbarColor(const std::optional<class ScrollbarColor> color)
```

Comment:
Passing by const copy is unnecessary, we should pass it by const reference (or move it)